Lib/idlelib/autocomplete_w.py

```diff
@@ -241,8 +243,10 @@ def winconfig_event(self, event):
         acw.wm_geometry("+%d+%d" % (new_x, new_y))
 
     def hide_event(self, event):
+        # This will be trigger when focus on widget or autocompletewindow
```

Comment:
Make proper docstring, which I believe 
"Hide autocomplete list if it exists and does not have focus."